src/main.js

```diff
@@ -0,0 +1,16 @@
+import indexPageModes from "./layouts/indexPageModes"
+import changeGameMode from "./changeGameMode"
+
+const indexPage = new indexPageModes()
+
+document.getElementById("main-grid-container").appendChild(indexPage.render())
+
+const btnChar = document.querySelector('.btn-characters')
+btnChar.addEventListener('click', ()=> changeGameMode('.main__header', '.main__description',"MODE: Who is this character?", 'You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select who from Star Wars is showed on the left (Jar Jar Binks right now) from available options.'))
```

Comment:
The arguments `text` and `description` share a lot of common content that could probably be customised with the use of of backticks and ${}. This could be done in this file by using additional variables or inside `changeGameMode`.